kythe/extractors/BUILD

```diff
@@ -51,7 +51,7 @@ proto_lang_toolchain(
         "@com_google_protobuf//:type_proto",
         "@com_google_protobuf//:wrappers_proto",
     ],
-    command_line = "--$(PLUGIN_OUT)=:$(OUT)",
+    command_line = "--PLUGIN_java_stubby_out=:%s",
```

Comment:
AFAIK, this should be `--PLUGIN_C++_out` (although actually *determining* that, without the aid of the `$(PLUGIN_OUT)` replacement) is incredibly challenging.  I had to run `bazel -s` to do so.